server/src/main/java/org/apache/iotdb/db/query/externalsort/ExternalSortJob.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright © 2019 Apache IoTDB(incubating) (dev@iotdb.apache.org)
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.iotdb.db.query.externalsort;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.iotdb.db.query.reader.IPointReader;
+
+/**
+ * This class represents an external sort job. Every job will use a separated directory.
+ */
+public class ExternalSortJob {
+
+  private long jobId;
+  private List<ExternalSortJobPart> partList;
+
+  public ExternalSortJob(long jobId, List<ExternalSortJobPart> partList) {
+    this.jobId = jobId;
+    this.partList = partList;
+  }
+
+  public List<IPointReader> executeWithGlobalTimeFilter() throws IOException {
+    List<IPointReader> readers = new ArrayList<>();
+    for (ExternalSortJobPart part : partList) {
+      readers.add(part.executeWithGlobalTimeFilter());
```

Comment:
I think `executeForIPointReader` is better than `executeWithGlobalTimeFilter`.

Reply:
fixed